surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/JUnit47ConcurrencyIT.java

```diff
@@ -32,14 +32,14 @@
  *
  * @author Kristian Rosenvold
  */
-public class Junit47concurrencyIT
```

Comment:
@britter I like the change :) and I appreciate it but the problem is with me last week I was unable to work in ASF.